### PR TITLE
Updates the supported operating system section of install guide

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
Puts the operating system versions we support in a table for better readability.

### Issues Resolved
Resolves https://github.com/opensearch-project/documentation-website/issues/4217

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@peterzhuamazon and @nateynateynate - please review for accuracy. Thanks!

File: _install-and-configure/install-opensearch/index.md [hunk 1]
```diff
@@ -17,7 +17,17 @@ This section details how to install OpenSearch on your host, including which ope
...
 
-We recommend installing OpenSearch on Red Hat Enterprise Linux (RHEL) or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). OpenSearch should work on most Linux distributions, but we only test a handful. We recommend RHEL 7 or 8, CentOS 7 or 8, Amazon Linux 2, or Ubuntu 16.04, 18.04, or 20.04 for any version of OpenSearch. OpenSearch also supports Windows Server 2019.
+OpenSearch is compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). While OpenSearch should work on most Linux distributions, we only test a subset. 
+
+The following table lists the operating system versions that we support. 
+
+OS | Version
+:---------- | :-------- 
+RHEL/CentOS |	7/8
+Rocky Linux / AlmaLinux |	8
```
Comment (peterzhuamazon) [inline _install-and-configure/install-opensearch/index.md:27]: Remove spaces?

File: _install-and-configure/install-opensearch/index.md [hunk 2]
```diff
@@ -17,7 +17,17 @@ This section details how to install OpenSearch on your host, including which ope
 
 ## Operating system compatibility
 
-We recommend installing OpenSearch on Red Hat Enterprise Linux (RHEL) or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). OpenSearch should work on most Linux distributions, but we only test a handful. We recommend RHEL 7 or 8, CentOS 7 or 8, Amazon Linux 2, or Ubuntu 16.04, 18.04, or 20.04 for any version of OpenSearch. OpenSearch also supports Windows Server 2019.
+OpenSearch is compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). While OpenSearch should work on most Linux distributions, we only test a subset. 
+
+The following table lists the operating system versions that we support.
```
Comment (peterzhuamazon) [inline _install-and-configure/install-opensearch/index.md:22]: `that we support` to `that we currently support`?

File: _install-and-configure/install-opensearch/index.md [hunk 3]
```diff
@@ -17,7 +17,17 @@ This section details how to install OpenSearch on your host, including which ope
 
 ## Operating system compatibility
 
-We recommend installing OpenSearch on Red Hat Enterprise Linux (RHEL) or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). OpenSearch should work on most Linux distributions, but we only test a handful. We recommend RHEL 7 or 8, CentOS 7 or 8, Amazon Linux 2, or Ubuntu 16.04, 18.04, or 20.04 for any version of OpenSearch. OpenSearch also supports Windows Server 2019.
+OpenSearch is compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). While OpenSearch should work on most Linux distributions, we only test a subset.
```
Comment (peterzhuamazon) [inline _install-and-configure/install-opensearch/index.md:20]: `OpenSearch is compatible` to `OpenSearch/OpenSearch-Dashboards are compatible?`

**Review by peterzhuamazon — APPROVED** [review]
Hu @hdhalter , have some comments, thanks.

File: _install-and-configure/install-opensearch/index.md (hunk 2 above)
Comment (hdhalter) [reply _install-and-configure/install-opensearch/index.md:22]: Curious - do you think we should backport this change all the way back to 1.3?

File: _install-and-configure/install-opensearch/index.md (hunk 1 above)
Comment (hdhalter) [reply _install-and-configure/install-opensearch/index.md:27]: You're right, the readability is better.

Comment (hdhalter) [reply _install-and-configure/install-opensearch/index.md:27]: Actually, according to our style guide, no spaces: When using a forward slash between words, do not insert space on either side of the slash. For example, AI/ML is correct whereas AI / ML is incorrect.

File: _install-and-configure/install-opensearch/index.md (hunk 2 above)
Comment (peterzhuamazon) [reply _install-and-configure/install-opensearch/index.md]: There are a few issues here:
1. This does not only apply to RPM related OS, but apply to all OS that does not support sha1 key by default, so I think the download page would be a better idea.
2. The download page yum/apt download link is not correctly link to rpm/deb page.
3. DEB page is missing in the 1.3 section of documentation website installation pages.

Comment (peterzhuamazon) [reply _install-and-configure/install-opensearch/index.md]: And yes I think we should also backport to 1.3, due to eventually 1.3 and 2.x might have different or same compatibilities that we dont know, and such a chart would help for sure.

**Comment by peterzhuamazon** [comment]
Hi @hdhalter please see my reply to your comments.

File: _install-and-configure/install-opensearch/index.md (hunk 2 above)
Comment (hdhalter) [reply _install-and-configure/install-opensearch/index.md]: Thanks, Peter. Updates to the following:
- I added a link from the RPM page to the operating system compatibility table
- I added redirects to the RPM and Debian pages so the "install" link on the Downloads page takes the user directly to those pages

Fanit is going to fix the issue with the parent link on the 1.3 version so we'll be able to backport easily to 1.3. Once she fixes that, I'll backport all installation files, including Debian. The 'copy' buttons should be appearing, as well. 

I'll create an issue to fix the Downloads page:
- Add a link to the "Operating System compatibility" chart on the install page

File: _install-and-configure/install-opensearch/index.md [hunk 4]
```diff
@@ -17,7 +17,17 @@ This section details how to install OpenSearch on your host, including which ope
 
 ## Operating system compatibility
 
-We recommend installing OpenSearch on Red Hat Enterprise Linux (RHEL) or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). OpenSearch should work on most Linux distributions, but we only test a handful. We recommend RHEL 7 or 8, CentOS 7 or 8, Amazon Linux 2, or Ubuntu 16.04, 18.04, or 20.04 for any version of OpenSearch. OpenSearch also supports Windows Server 2019.
+OpenSearch and OpenSearch Dashboards are compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). While OpenSearch and OpenSearch Dashboards should work on most Linux distributions, we only test a subset.
```
Comment (kolchfa-aws) [inline _install-and-configure/install-opensearch/index.md:20]:
```suggestion
OpenSearch and OpenSearch Dashboards are compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [`systemd`](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, and Ubuntu Long-Term Support (LTS). While OpenSearch and OpenSearch Dashboards should work on most Linux distributions, we only test a subset. 
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Pull request merged**